src/core/include/units/magnitude.h

```diff
@@ -104,9 +104,9 @@ base_power(T) -> base_power<int>;
 // Implementation for BasePower concept (below).
 namespace detail {
 template<typename T>
-static constexpr bool is_base_power = false;
+constexpr bool is_base_power = false;
```

Comment:
```suggestion
inline constexpr bool is_base_power = false;
```

`static` did not have much sense indeed. `inline` needed to prevent ODR violation.

Reply:
Is it relevant to supported compilers? See https://github.com/cplusplus/draft/pull/4625.

Reply:
Hmm, I am not sure. Maybe you are right. Although the C++ standard is explicit that all variable templates should be implemented as here: https://eel.is/c++draft/meta.type.synop.